test-support/page-object/properties/collection.js

```diff
@@ -0,0 +1,76 @@
+import Ember from 'ember';
+import Ceibo from 'ceibo';
+import { buildSelector } from '../helpers';
+import { create } from '../create';
+import { count } from './count';
+
+function merge(target, ...objects) {
+  objects.forEach(o => Ember.merge(target, o));
+
+  return target;
+}
+
+function select(target, ...keys) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["select" is defined but never used](https://www.codacy.com/app/san650/ember-cli-page-object/file/1957769446/issues/source?bid=2903707&fileBranchId=3076256#l13)